index.html

```diff
@@ -29,5 +31,153 @@ <h4>LET'S CONNECT</h4>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="button" type="button">See Project</button>
+            </article>
+        </div>
+        <div class="work-card">
+            <img class="profile-image" src="images/gymfit.png" alt="illustration of our gym activities" width="295"
+                height="220">
+            <article class="client-info">
+                <h2 class="project-title">Tonic</h2>
+                <ul class="credentials-list">
+                    <li class="canopy">CANOPY</li>
+                    <li><i class="fa fa-circle fa-sm" aria-hidden="true"></i></li>
+                    <li>Back End Dev</li>
+                    <li><i class="fa fa-circle fa-sm" aria-hidden="true"></i></li>
+                    <li>2015</li>
+                </ul>
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="language-list">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="button" type="button">See Project</button>
+            </article>
+        </div>
+        <div class="work-card">
+            <img class="profile-image" src="images/art-printing.png" alt="professional art printing" width="295"
+                height="220">
+            <article class="client-info">
+                <h2 class="project-title">Mult-Post Stories</h2>
+                <ul class="credentials-list">
+                    <li class="canopy">CANOPY</li>
+                    <li><i class="fa fa-circle fa-sm" aria-hidden="true"></i></li>
+                    <li>Back End Dev</li>
+                    <li><i class="fa fa-circle fa-sm" aria-hidden="true"></i></li>
+                    <li>2015</li>
+                </ul>
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="language-list">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="button" type="button">See Project</button>
+            </article>
+        </div>
+    </section>
+        <!-- about myself start-->
+        <section class="about-myself">
+            <h2 class="about-myself-title">About <br> Myself</h2>
+            <p class="my-profile">
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <a class="contact">LET'S CONNECT</a>
```

Comment:
- [ ] Your about section looks good however not perfect yet 😉. Here instead of a, you should use one headings in HTML. Maybe you can use this code as an example
```html
<h3 class="contact">LET'S CONNECT</h3>
```

Reply:
Ok I will use heading in HTML.